internal/state/change_processor_test.go

```diff
@@ -350,11 +418,16 @@ var _ = Describe("ChangeProcessor", func() {
 					Routes: map[types.NamespacedName]*graph.Route{
 						{Namespace: "test", Name: "hr-1"}: expRouteHR1,
 					},
+					Secrets: map[types.NamespacedName]*graph.Secret{},
+				}
+
+				emptyGraph = &graph.Graph{
+					Secrets: map[types.NamespacedName]*graph.Secret{},
```

Comment:
Why is Secrets the only map that gets initialized even when Graph is empty? Can this be avoided to just have it be nil? (not a huge deal, more of just consistency with the other resources)

Reply:
I can make it nil 👍